Htmlcss ex

julianpoma opened the pull request:
Hi Maxi.

I created this new branch in order to get all the excersises organized.

I fixed all your previous corrections on the HTML excersises, and i added the CSS ones. 

See you tomorrow!

maximilianotulian commented on 00-HTML-CSS/Ex-2.5.2/styles.css:
```diff
@@ -0,0 +1,70 @@
+  height: 100%;
+}
+.screen {
+
```
Minor thing, remove this blank line

maximilianotulian commented on 00-HTML-CSS/Ex-2.5.2/styles.css:
```diff
@@ -0,0 +1,70 @@
+    -o-transition: all 0.3s ease-in-out;
+    -ms-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    width: 100%;
```
Please remove the style duplicated

maximilianotulian commented on 00-HTML-CSS/Ex-2.5.2/styles.css:
```diff
@@ -0,0 +1,70 @@
+}
+
+.clearfix {
+  *zoom: 1;
```
Interesting, `*`

maximilianotulian commented on 00-HTML-CSS/Ex-2.5.2/index.html:
```diff
@@ -0,0 +1,47 @@
+</head>
+<body>
+    <div class="container">
+        <div class="row clearfix">
```
We missed the definition of `row` in the css

julianpoma commented:
hi maxi

"The next time try to not use style attributes inside the html elements."
Fixed

"The for attribute should point to an element id"
Ok. I though it was the label identifier, that lets you access and modify its value. Like on C#.

"This is a warning. The normalize should be required first. Anyway you did it correctly in the following exercise."
Fixed and thanks!

"The last two selectors have an addition tab. Try to keep consistency across the files."
Noted!

"There is not need to use !important. Remember to keep the specificity as lower as possible."
mistype there. I was trying things.

"Why we're using double qoutes? If you want to highlight the sentence is right, the other way we should remove it."
Another mistype. Fast copy and paste!

"Question, what happen if I change the float from right to left?"
It will move to the left and be next to the previous box. It will go down if it hasnt got any space left.

"Whenever we use an img tag, we should add an alternative description"
Roger !

"Copy and paste is really useful, just be aware of its side effects. Please check the class of the second <p>"
Fixed! Missed " "

"Please remove the style duplicated"
Removed

"We missed the definition of row in the css"
Removed!

maximilianotulian commented on 00-HTML-CSS/Ex-1/signup.html:
```diff
@@ -8,15 +8,15 @@
+      <label for="">Birthday:</label><br>
       <input type="date" name="birth" placeholder="dd/mm/aaaa"><br><br> <!-- doesnt work on firefox, thats why the placeholder -->
-      <label for="sport">Favourite sport:</label><br>
+      <label for="">Favourite sport:</label><br>
```
Maybe I make myself unclear, the label should point to tag id. For example, the first label you have is for "fname". The input first name should have the id="fname". Anyway is something minor, not needed to change it. Remember it for the next ocassion.

julianpoma replied:
Ok maxi, thank you and got it. Can we merge this branch with master?
